dashboard/modules/stats_collector/stats_collector_head.py

```diff
@@ -144,17 +207,78 @@ async def _update_node_stats(self):
                 continue
             try:
                 reply = await stub.GetNodeStats(
-                    node_manager_pb2.GetNodeStatsRequest(), timeout=2)
+                    node_manager_pb2.GetNodeStatsRequest(
+                        include_memory_info=self._collect_memory_info),
+                    timeout=2)
                 reply_dict = node_stats_to_dict(reply)
                 DataSource.node_stats[node_id] = reply_dict
             except Exception:
                 logger.exception(f"Error updating node stats of {node_id}.")
 
+    async def _update_log_info(self):
+        aioredis_client = self._dashboard_head.aioredis_client
+        receiver = Receiver()
+
+        channel = receiver.channel(ray.gcs_utils.LOG_FILE_CHANNEL)
+        await aioredis_client.subscribe(channel)
+        logger.info("Subscribed to %s", channel)
+
+        async for sender, msg in receiver.iter():
+            try:
+                data = json.loads(ray.utils.decode(msg))
+                logger.error(f"data={data}")
+                ip = data["ip"]
+                pid = str(data["pid"])
```

Comment:
Why convert the `pid` to str? Is it to be consistent with the `ip_and_pid_to_errors`?